backends/dart_minerva/lib/builders/endpoints_builder.dart

```diff
@@ -0,0 +1,40 @@
+import 'dart:convert';
+
+import 'package:minerva/minerva.dart';
+
+class EndpointsBuilder extends MinervaEndpointsBuilder {
+  @override
+  void build(Endpoints endpoints) {
+    endpoints.get('/', _root);
+
+    endpoints.post('/echo', _echo);
+
+    endpoints.post('/file_upload', _fileUpload);
+
+    endpoints.post('/json_obj', _jsonObject);
+  }
+
+  dynamic _root(ServerContext context, MinervaRequest request) {
+    return {'response': 'Hello, world!'};
+  }
+
+  dynamic _echo(ServerContext context, MinervaRequest request) async {
+    var json = await request.body.asJson();
+
+    return {"response": "Hello, ${json['name']}!"};
+  }
+
+  dynamic _fileUpload(ServerContext context, MinervaRequest request) async {
+    var form = await request.body.asForm();
+
+    var field = form.data['benchmark'] as FormDataFile;
+
+    return utf8.decode(field.bytes);
+  }
+
+  dynamic _jsonObject(ServerContext context, MinervaRequest request) async {
+    var json = jsonDecode(await request.body.asText());
```

Comment:
You can use your `asJson` method here as well

Reply:
Yes, it is. But there was a little confusion with this method (asJson), or rather how it worked.

In general, I will redo it so that it is used in the test.